### DISCUSSION TITLE
Broken installation?

### DISCUSSION THREAD

**Discussion by stathi02** [PMEAL/OpenPNM#2611] [Q&A]
Hello, 
i am trying to run the first basic example:

```
import openpnm as op

# Define geometrical parameters
Lc = 1e-4
Nx, Ny, Nz = (10, 10, 10)

# Generate network, geometry, phase, and physics
pn = op.network.Cubic(shape=[Nx, Ny, Nz], spacing=Lc)
geo = op.geometry.SpheresAndCylinders(network=pn, pores=pn.Ps, throats=pn.Ts)
Hg = op.phases.Mercury(network=pn)
phys = op.physics.Standard(network=pn, phase=Hg, geometry=geo)

# Create algorithm and run simulation
mip = op.algorithms.Porosimetry(network=pn, phase=Hg)
mip.set_inlets(pores=pn.pores(['left', 'right', 'top', 'bottom']))
mip.run()
```

I am getting the following error:

runfile('C:/openpnm test/simple.py', wdir='C:/openpnm test')
Traceback (most recent call last):

  File C:\openpnm test\simple.py:14 in <module>
    geo = op.geometry.SpheresAndCylinders(network=pn, pores=pn.Ps, throats=pn.Ts)

AttributeError: module 'openpnm' has no attribute 'geometry'

Is there any way to fix it?

Python 3.9.7,

**Comment by jgostick** [comment]
Hi @stathi02 
I think your best best is to just reinstall anaconda and openpnm in a fresh install.
